Cargo.toml

```diff
@@ -30,6 +30,7 @@ tokio = { version = "1.34.0", features = ["macros", "rt"] }
 prefixed-api-key = { version = "0.1.0", features = ["sha2"]}
```

Comment:
Documentation: Worth also updating the HTTP and CLI examples in the README.md?

Reply:
Done! :)

Reply:
Thank you!